.github/workflows/deploy.yml

```
@@ -50,43 +51,70 @@ jobs:
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
-          aws-region: us-east-1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
 
       - name: Login to Amazon ECR Public
-        id: login-ecr-public
-        uses: aws-actions/amazon-ecr-login@v2
+        uses: docker/login-action@v3
+        with:
+          registry: public.ecr.aws
+          region: us-east-1
```

actionlint (1.7.7)

input "region" is not defined in action "docker/login-action@v3". available inputs are "ecr", "logout", "password", "registry", "username"

(action)